### PR TITLE
Added TMVELOXF411 unified target

### PR DESCRIPTION
**Pull request by Guidus93**
@haslinghuis here the PR for the TMotor Velox F411 with correct gyro orientation.

### CODE REVIEW

File: configs/default/TMTR-TMVELOXF411.config [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin B06 0
+# pin B06: DMA1 Stream 0 Channel 2
+dma pin B07 0
+# pin B07: DMA1 Stream 3 Channel 2
+dma pin A08 0
+# pin A08: DMA2 Stream 6 Channel 0
+
+# feature
+feature -RX_PARALLEL_PWM
```
Comment (haslinghuis) [inline configs/default/TMTR-TMVELOXF411.config:68]: Please omit. RX_SERIAL is default now.

File: configs/default/TMTR-TMVELOXF411.config [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+set ibata_scale = 250
+set beeper_inversion = ON
+set beeper_od = OFF
+set system_hse_mhz = 8
+set max7456_spi_bus = 2
+set flash_spi_bus = 3
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set gyro_1_sensor_align = CW270
+set gyro_1_align_yaw = 2700
```
Comment (haslinghuis) [inline configs/default/TMTR-TMVELOXF411.config:93]: Please add new line

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: configs/default/TMTR-TMVELOXF411.config (hunk 1 above)
Comment (Guidus93) [reply configs/default/TMTR-TMVELOXF411.config]: Done

File: configs/default/TMTR-TMVELOXF411.config (hunk 2 above)
Comment (Guidus93) [reply configs/default/TMTR-TMVELOXF411.config]: Done

**Review by haslinghuis — APPROVED** [review]

**Review by KarateBrot — APPROVED** [review]

**Comment by liu-root** [comment]
I'll test this on weekend just to confirm.

**Pull request merged**